src/sql/workbench/contrib/queryplan2/browser/media/queryPlan2.css

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.qp2-actionbar-container {
+	height: 100%;
+	width: 30px;
+	position: absolute;
+	top: 0px;
+	right: 0px;
+}
+
+.qp2-container {}
+
+.query-plan2-container {
+	width: 100%;
+	height: 500px;
+	position: relative;
+	border-top: 1px solid;
+}
+
+.properties-container {
+	position: absolute;
+	top: 0px;
+	right: 32px;
+	height: 100%;
+	overflow-y: scroll !important;
```

Comment:
what are we trying to overwrite here with the important? you can make this selector more specific to avoid needing to do so.